homeassistant/components/habitica/const.py

```diff
@@ -7,8 +7,11 @@
 DEFAULT_URL = "https://habitica.com"
 DOMAIN = "habitica"
 
+# service constants
 SERVICE_API_CALL = "api_call"
 ATTR_PATH = CONF_PATH
 ATTR_ARGS = "args"
-ATTR_DATA = CONF_SERVICE_DATA
+
+# event constants
 EVENT_API_CALL_SUCCESS = f"{DOMAIN}_{SERVICE_API_CALL}_success"
+ATTR_DATA = CONF_SERVICE_DATA
```

Comment:
```suggestion
ATTR_DATA = "data"
```

Reply:
Ah. I thought this might be the case what you wanted. Will do now